src/metadata/ActionMetadata.ts

```diff
@@ -256,7 +256,7 @@ export class ActionMetadata {
     
     static appendBaseRouteToRegexpRoute(route: RegExp, baseRoute: string) {
         if (!baseRoute || baseRoute === "") return route;
-        const fullPath = baseRoute.replace("\/", "\\\\/") + route.toString().substr(1);
+        const fullPath = `^${baseRoute}${route.toString().substr(1)}?$`;
```

Comment:
@19majkel94 Do you mean as comments on code itself for future references or just here on the PR?

I am asking because I'd rather it on the code, I got very confused with the original `.replace("\/", "\\\\/")` that seemed to be not doing anything